the-journal-of-foot-and-ankle-surgery.csl

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>The Journal of Foot & Ankle Surgery</title>
```

Comment:
http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2FPOBrien333%2Fstyles%2Ff74f241a51ada8a3057f7de7ca8836eb194ec49a%2Fthe-journal-of-foot-and-ankle-surgery.csl&version=1.0.1

Yes, that's it because the validator can't parse the whole file (since it's not valid XML). It still does a decent job pinpointing the problem, though.